### PR TITLE
Sterilizes the codebase - Pt.6 Dwarfism now permanently gives you dwarf beard

### PR DESCRIPTION
**Pull request by JamieD1**
Dwarfism now permanently gives you dwarf beard

:cl:  
rscadd: Dwarfism now permanently gives you dwarf beard
/:cl:

### CODE REVIEW

**Comment by Smudged** [comment]
Dwarfism shouldn't give you a beard. It's bad.

**Comment by VaelophisNyx** [comment]
If this is about people using dwarfism to RP as a child

Admin. Issue.

**Comment by VaelophisNyx** [comment]
That said if it ISN'T
I'm probably fine with it, but expect dwarfism to be a mutation no-one ever takes

**Comment by GraveHat** [comment]
> That said if it ISN'T I'm probably fine with it, but expect dwarfism to be a mutation no-one ever takes

This wouldn't affect any races that can't get beards so there shouldn't be much of a difference over there.

**Comment by SomeguyManperson** [comment]
Don't think this is required if people want to rp as a dwarf they can get their own beard

**Comment by Smudged** [comment]
> If this is about people using dwarfism to RP as a child
> 
> Admin. Issue.

I will also note, Only 1 person ever did this, it's simply a non-issue.

**Comment by Anvilman6** [comment]
Rocking and stoning

**Comment by Marmio64** [comment]
tactical beardification of the beardless

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -96,7 +96,18 @@
...
+	grow_beard(owner)
+
+/datum/mutation/human/dwarfism/proc/grow_beard(mob/living/carbon/human/owner)
+	var/mob/living/carbon/human/H = owner
+	to_chat(H, span_warning("Your chin itches."))
+	H.facial_hair_style = "Beard (Dwarf)"
+	H.update_hair()
+
+/datum/mutation/human/dwarfism/on_life()
+	if(owner.facial_hair_style == "Shaved")
```
Comment (JamieD1) [inline code/datums/mutations/body.dm]:
```suggestion
	if(owner.facial_hair_style != "Beard (Dwarf)")
```

**Review by ToasterBiome — APPROVED** [review]

**Comment by SomeguyManperson** [comment]
do not.

**Comment by Sniblet** [comment]
Hehe funny sabotage people's appearance take that "people" who care about cosmetics lol

**Comment by KillerOrcaCora** [comment]
ROCK AND STONE!!!!!!!!!!!!!!!!!!!!

**Pull request merged**